src/plugins/condition/components/Condition.vue

```diff
@@ -256,6 +279,9 @@ export default {
                 index: Number(ev.target.closest('.widget-condition').getAttribute('data-condition-index'))
             });
         },
+        updateCurrentCondition() {
```

Comment:
We no longer need this either. I think it has been removed in the topic-conditionals branch.
Can you do a compare with topic-conditionals to make sure?

Reply:
Done